templates/resources/metrics.md

```diff
@@ -3,28 +3,26 @@ title: {Service/Controller name}
 type: Metrics
```

Comment:
The document could be removed or archived

Reply:
Removed :)